documentation/doc-Migration_Toolkit_for_Virtualization/modules/selecting-migration-network-for-target-provider.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="selecting-migration-network_{context}"]
+= Selecting a migration network for {a-virt} provider
+
+You can select a default migration network for {a-virt} provider in the {project-short} web console to improve performance. The default migration network is the `pod` network, which is not optimal for disk transfer.
```

Comment:
"which **may** not be optimal for disk transfer"

We don't know if the pod network is fine or not.